spec/automation/unit/method_validation/quota_source_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+include QuotaHelper
+
+def run_automate_method(provision_request)
+  attrs = []
+  attrs << "MiqProvisionRequest::miq_provision_request=#{@miq_provision_request.id}&" \
```

Comment:
@tinaafitz 
@miq_provision_request should be provision_request since we are passing it in